spec/src/main/asciidoc/core/implementation.asciidoc

```diff
@@ -30,10 +30,10 @@ If the managed bean class is a generic type, it must have scope `@Dependent`. If
 
 ==== Which Java classes are managed beans?
 
-A top-level Java class is a managed bean if it meets all of the following conditions:
+A Java class is a managed bean if it meets all of the following conditions:
 
-* It is not a non-static inner class.
-* It is a concrete class, or is annotated `@Decorator`.
+* It is not a non-static nested class.
```

Comment:
This doesn't sound quite right.  Maybe its current spec behavior, but I would expect that static, not 'non-static' classes are eligible. Or maybe the sentence should be rephrased as

`It is a static nested class`